### ISSUE TITLE
Comma-first (return) explists with comments are broken

### ISSUE THREAD

**Issue by appgurueu** [JohnnyMorganz/StyLua#662]
Run the following through StyLua (version 0.14.3):

```lua
function f()
	return a -- adoc
		, b -- bdoc
end
```

result:

```lua
function f()
	return a -- adoc,
		b -- bdoc
end
```

**This is a syntax error.**

The comma is correctly (consistent with paramlists) moved to `a`, but instead of being moved to right after `a`, it is moved to after the comment.

Related: https://github.com/JohnnyMorganz/StyLua/issues/307. This is essentially the same issue but for (`retstat`?) `explists`.

**Comment by JohnnyMorganz** [comment]
Also an issue in a local assignment
```lua
local a -- adoc
, b -- bdoc
= 1, 2
```

**Comment by JohnnyMorganz** [comment]
This has been fixed and is available in v0.17.1

**Comment by appgurueu** [comment]
Thanks for the quick fix!
